yocto/x86/metas_ids

```diff
@@ -7,5 +7,5 @@ meta-openembedded/meta-filesystems
 meta-virtualization
 meta-selinux
 meta-java
-meta-trustx
 meta-trustx-intel
+meta-trustx
```

Comment:
order?

Reply:
yocto seems to sort layers solely based on priority,
this was just a cosmetic change to reflect the layer priority